samples/zigbee/ncp/src/main.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/** @file
+ *  @brief Zigbee Network Co-processor sample
+ */
+
+#include <drivers/uart.h>
+#include <usb/usb_device.h>
+#include <logging/log.h>
+#include <zb_nrf_platform.h>
+#include <zb_osif_ext.h>
+
+LOG_MODULE_REGISTER(app);
+
+int main(void)
+{
+	LOG_INF("Starting Zigbee Network Co-processor sample");
+
+#ifdef CONFIG_USB
+	/* Enable USB device. */
+	int ret = usb_enable(NULL);
+
+	if (ret != 0)
+		return ret;
+
+#ifdef CONFIG_UART_LINE_CTRL
+	const struct device *uart_dev;
+	uint32_t dtr = 0U;
+
+	uart_dev = device_get_binding(CONFIG_ZIGBEE_UART_DEVICE_NAME);
+	if (uart_dev == NULL) {
+		while (1)
```

Comment:
This seems like a critical error, perhaps `LOG_ERR` some message here and exit instead of busy looping?

Reply:
changed to `LOG_ERR` followed by `return -EIO`